ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -576,6 +576,11 @@
           + "Default is false. If false, persistent queries will use separate "
           + " runtimes, if true, new queries may share streams instances.";
 
+  public static final String KSQL_NEW_QUERY_PLANNER_ENABLED =
+      "ksql.new.query.planner.enabled";
```

Comment:
Shouldn't we have a good config name instead of `new.query.planner`? Just wondering what if we add an new improved planner after this new planner was added :P?

Reply:
Happy to use a different name if you have a suggestion. But the config should be temporary only anyway.